packages/amazon-bedrock/src/bedrock-provider.ts

```diff
@@ -85,15 +82,16 @@ export interface AmazonBedrockProvider extends ProviderV2 {
 
   embedding(modelId: BedrockEmbeddingModelId): EmbeddingModelV2<string>;
 
-  image(
-    modelId: BedrockImageModelId,
-    settings?: BedrockImageSettings,
-  ): ImageModelV2;
+  /**
+Creates a model for image generation.
+@deprecated Use `imageModel` instead.
+   */
+  image(modelId: BedrockImageModelId): ImageModelV2;
```

Comment:
`image` is deprecated, right? Is saw it in one of the models and applied it everywhere. Shall we remove it as part of v5 (in another PR?)